src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -344,6 +344,7 @@ def test_unload_plugin(device):
 
 
 @pytest.mark.template_extension()
+@pytest.mark.dynamic_library()
```

Comment:
@jiwaszki please, check changes in python tests.

Reply:
It's valid change, resolving convo.